### DISCUSSION TITLE
Markup Support

### DISCUSSION THREAD

**Discussion by henry74** [flyerhq/flutter_chat_ui#427] [Ideas & Requests] [answered]
With the advent of ChatGPT, which provides some responses using markup language, it would be beneficial for the library to fully support markup.  I definitely see there is some support for markup (bold, etc) but not for code (e.g. ```).  Thoughts?

**Comment by demchenkoalex** [comment, marked as answer]
part of v2 https://github.com/flyerhq/flutter_chat_ui/issues/646
